web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -280,41 +315,20 @@ export function ProjectCard({
                       {containedButton.text}
                     </ContainedButton>
                   )}
-                  {(onButtonClick || isPrefinanceProject || offChain) && (
-                    <OutlinedButton
-                      onClick={event => {
-                        event.stopPropagation();
-                        if (asAdmin) {
-                          onButtonClick && onButtonClick();
-                        } else if (isPrefinanceProject) {
-                          window.open(
-                            projectPrefinancing?.stripePaymentLink,
-                            '_newtab',
-                          );
-                        } else if (offChain) {
-                          onClick && onClick();
-                        } else {
-                          track &&
-                            track<Buy1Event>('buy1', {
-                              url: pathname ?? '',
-                              cardType: 'project',
-                              buttonLocation: 'projectCard',
-                              projectName: name,
-                              projectId: id,
-                              creditClassId,
-                            });
-                          onButtonClick && onButtonClick();
-                        }
-                      }}
-                      size="small"
-                      startIcon={buttonStartIcon}
-                      disabled={isButtonDisabled}
-                      sx={{ width: '100%' }}
-                      className={buttonClassName}
-                    >
-                      {buttonText}
-                    </OutlinedButton>
-                  )}
+                  {(onButtonClick || isPrefinanceProject || offChain) &&
+                    (isButtonDisabled ? (
```

Comment:
like I said above, since this button is not always a "create post" button, always displaying the `CREATE_POST_DISABLED_TOOLTIP ` tooltip when it's disabled does not make sense.
Check now what it does on the /projects page for instance:

![image](https://github.com/user-attachments/assets/1fca03dd-047b-481f-b59b-31bed8496243)

I believe we should rather provide the tooltip text as a prop and if that's defined, we display the tooltip